cmd/juju/commands/switch.go

```diff
@@ -5,10 +5,10 @@ package commands
 
 import (
 	"fmt"
-	"os"
-
 	"github.com/juju/cmd/v4"
 	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+	"os"
```

Comment:
The `os` package should go in the first grouping of imports (std lib).

Reply:
Sure I didn't gci files. Fixed.